### DISCUSSION TITLE
Twitter internal API

### DISCUSSION THREAD

**Discussion by fa0311** [fa0311/twitter-openapi#44] [Announcements]
### Twitter API

Python
https://github.com/fa0311/twitter_openapi_python
Dart
https://github.com/fa0311/twitter_openapi_dart
TypeScript
https://github.com/fa0311/twitter-openapi-typescript

### Twitter Login (web driver)
Python
https://github.com/tsukumijima/tweepy-authlib (Recommend)
https://github.com/fa0311/TwitterFrontendFlow

### Twitter Login (web driver)
Flutter
https://github.com/fa0311/twitter_openapi_dart/tree/main/twitter_openapi_dart#login

### AwesomeList
https://github.com/fa0311/AwesomeTwitterUndocumentedAPI

### Twitter Strange Specs
Count does not limit number of tweet results <https://github.com/fa0311/twitter-openapi-typescript/issues/58>
Timelines eventually stop returning posts <https://github.com/fa0311/TwitterInternalAPIDocument/issues/550>

### Deprecated

Login using v1.1 login endpoint.
Use tokens obtained by the official API on web endpoints.
Use tokens obtained by WebAPI with official endpoints.
Use old endpoints that are not in use. (Specifically v1, v1.1, and v2, but not all, since some are still in use).

---
If you have any additional information, please write a comment.

**Comment by ghost** [comment]
How i can set proxy to twitter client? this script lacks the ability to use proxies :(
